src/App.js

```diff
@@ -26,14 +27,15 @@ function App({ currentUser = false }) {
         <Navbar />
         <Switch>
           <Route exact path="/" component={Dashboard} />
+          <Route exact path="/connect" component={Integrate} />
           <Route exact path="/flow/create" component={CreateFlow} />
           <Route exact path="/onboard" component={Onboard} />
           <Route
             exact
             path="/auth"
-            render={() => (currentUser ? <Redirect to="/" /> : <LoginForm />)}
+            children={() => (currentUser ? <Redirect to="/" /> : <LoginForm />)}
           />
-          <Route exact path="/connect" component={Integrate} />
+          <Route exact path="/athr/:service" component={CodeHandler} />
```

Comment:
Lets make this `/authorize/:service` which would make it easier to understand the function of api.